client/my-sites/domains/domain-management/components/transfer-connected-domain-nudge/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { Icon, starFilled } from '@wordpress/icons';
+import { useTranslate } from 'i18n-calypso';
+import moment from 'moment';
+import { useMyDomainInputMode } from 'calypso/components/domains/connect-domain-step/constants';
+import { domainUseMyDomain } from 'calypso/my-sites/domains/paths';
+import type { TransferConnectedDomainNudgeProps } from './types';
+import './style.scss';
+
+const TransferConnectedDomainNudge = ( {
+	siteSlug,
+	domain,
+}: TransferConnectedDomainNudgeProps ): JSX.Element | null => {
+	const translate = useTranslate();
```

Comment:
I think we should also check the `domain.Type` here to make sure it's `'MAPPED'` before we attempt to display anything.  I know that we're not "supposed" to be sending the registry expiry date for registered domains, but an inadvertent change on the back-end could easily cause us to mistakenly display this message for registered domains.

Reply:
Agreed, I think that's a good idea 👍